deltachat-ios/Controller/AccountSetup/AccountSetupController.swift

```diff
@@ -729,3 +727,21 @@ extension AccountSetupController: UITextFieldDelegate {
         }
     }
 }
+
+extension AccountSetupController: CertificateCheckDelegate {
+    func onCertificateCheckChanged(newValue: Int) {
+        certValue = newValue
+    }
+}
+
+class AccountSetupSecurityValue: SecuritySettingsDelegate {
+    var value: Int
+
+    init(initValue: Int) {
+        value = initValue
+    }
+
+    func onSecuritySettingsChanged(newValue: Int) {
+        value = newValue
+    }
+}
```

Comment:
What's the reason this implementation of `SecuritySettingsDelegate` is a dedicated class, while `CertificateCheckDelegate` is implemented as extension of `AccountSetupController`?

Reply:
we need two different delegates for `SecuritySettingsDelegate` - i did not find out a smart way to do that as an extension. one could add some state to the delegate - but that seems to work against the idea of the delegate